api/tools/src/main/java/org/pmiops/workbench/tools/WorkspaceExportRow.java

```diff
@@ -38,7 +50,7 @@ public void setCreatorFirstSignIn(String creatorFirstSignIn) {
   }
 
   @CsvBindByName(column = "First Sign In")
-  @CsvBindByPosition(position = 2)
+  @CsvBindByPosition(position = 3)
```

Comment:
welp this is certainly tedious.  assuming there's no better way or you'd have done it?

Reply:
Indeed - I didn't do the initial implementation or review, but from a quick scan around some of the documentation, it seems that custom names + custom ordering is not very well supported by this CSV library. I would not be surprised if this is the best way (at least, for this library). Will consider digging deeper if we decide to extend the lifespan of this tool further - it may be going away soon'ish with new metrics export plans.

Reply:
FWIW, this was not intended to be a long-lived implementation.